src/components/WeeklyInfo.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import Chart from 'react-apexcharts';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.25.3/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module 'react-apexcharts'.